lib/gis/colors.desc

```diff
@@ -48,6 +48,7 @@ sepia: yellowish-brown through to white
 slope: r.slope.aspect-type slope colors for raster values 0-90
 soilmoisture: soilmoisture color table (0.0-1.0)
 srtm: color palette for Shuttle Radar Topography Mission elevation
+srtm_pct: modified color palette for Shuttle Radar Topography Mission using relative elevation (0-100%) within a map instead of absolute elevation in meters
```

Comment:
Just to keep the description length consistent, it should be shorter, something like:
```suggestion
srtm_pct: relative color palette for Shuttle Radar Topography Mission
```
notice in r.colors man page it will add "[range: map values]" automatically